### PR TITLE
add lexer for the Lasso language

### PR DESCRIPTION
**Pull request by EricFromCanada**
See a demo here: http://ericfromcanada.bitbucket.org/javascript/demo-lasso.html
This closes issue #311.

### CODE REVIEW

File: src/lang-lasso.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          // single-line or block comments
+          [PR['PR_COMMENT'],      /^\/\/[^\r\n]*|\/\*[\s\S]*?\*\//],
+          // member variables or keyword parameters
+          [PR['PR_ATTRIB_NAME'],  /^-(?!infinity)[a-z_][\w.]*|\.\s*'[a-z_][\w.]*'/i],
+          // numeral as decimal or scientific notation
+          [PR['PR_LITERAL'],      /^\d*\.\d+(?:e[-+]?\d+)?|infinity\b|NaN\b/i],
+          // tag literals
+          [PR['PR_ATTRIB_VALUE'], /^::\s*[a-z_][\w.]*/i],
+          // constants
+          [PR['PR_LITERAL'],      /^(?:true|false|none|minimal|full|all|void|and|or|not|bw|nbw|ew|new|cn|ncn|lt|lte|gt|gte|eq|neq|rx|nrx|ft)\b/i],
```
Comment (mikesamuel) [inline src/lang-lasso.js]: Are keywords really case-insensitive?

Comment (EricFromCanada) [inline src/lang-lasso.js]: Yes, all Lasso keywords and variables are case-insensitive.

**Comment by mikesamuel** [comment]
Great.  Could you rebase and I'll merge?

**Comment by EricFromCanada** [comment]
Rebased, should be good to go.

**Pull request merged**